docs/UserGuide.md

```diff
@@ -53,8 +53,46 @@ burger | 23 Sep 2021
 ____________________________________________________
 ```
 
+### Remove an item from Fridget: `remove`
+Remove an item from Fridget's ledger.
 
-### Add an item into Fridget: `list`
+Format: remove <ITEM_NAME>
+
+* The `ITEM_NAME` can be in a natural language format.
+* The `ITEM_NAME` should be the same or be a subset of the name of the item you are trying to delete.
+
```

Comment:
If im not wrong, if u have multiple qty, then they will prompt how many to remove